pkg/javadb/client.go

```diff
@@ -85,6 +85,20 @@ func (u *Updater) Update() error {
 	return nil
 }
 
+func (u *Updater) isNewDB(meta db.Metadata) bool {
+	now := time.Now().UTC()
+	if now.Before(meta.NextUpdate) {
+		log.Debug("Java DB update was skipped because the local Java DB is the latest")
+		return true
+	}
+
+	if now.Before(meta.DownloadedAt.Add(time.Hour * 24)) { // 1 day
+		log.Debug("Java DB update was skipped because the local Java DB was downloaded during the last hour")
```

Comment:
Log message should be updated, (day, not hour)

Reply:
Nice catch. I overlooked it. Thanks!

Reply:
Thanks!
Updated in https://github.com/aquasecurity/trivy/pull/7592/commits/1de0a8177759c428dc7e371d9b467c310eb00dc8